File: examples/python/mandelbrot.py
def mandelbrot():
    width = 800
    height = 800
    max_iteration = 1000
    x_min = -2.5
    x_max = 1.0
    y_min = -1.25
    y_max = 1.25

    for y in range(height):
        x = 0
        for x in range(width):
            cx = x_min + (x * (x_max - x_min) / width)
            cy = y_min + (y * (y_max - y_min) / height)
            zx = 0.0
            zy = 0.0
            iteration = 0
            while (zx * zx + zy * zy < 4.0) and (iteration < max_iteration):
                xtemp = zx * zx - zy * zy + cx
                zy = 2.0 * zx * zy + cy
                zx = xtemp
                iteration = iteration + 1

mandelbrot()

File: examples/javascript/mandelbrot.js
const width = 800;
const height = 800;
const maxIter = 1000;

const xMin = -2.5;
const xMax = 1.0;
const yMin = -1.25;
const yMax = 1.25;

for (let y = 0; y < height; y++) {
	for (let x = 0; x < width; x++) {
		const cx = xMin + (x * (xMax - xMin)) / width;
		const cy = yMin + (y * (yMax - yMin)) / height;

		let zx = 0.0;
		let zy = 0.0;

		let iter = 0;

		while (zx * zx + zy * zy < 4.0 && iter < maxIter) {
			const xtemp = zx * zx - zy * zy + cx;

			zy = 2.0 * zx * zy + cy;
			zx = xtemp;

			iter++;
		}
	}
}
